core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BpmBillingProjectLifecycle.scala

```diff
@@ -22,19 +28,20 @@ import org.broadinstitute.dsde.rawls.model.{
 
 import java.util.UUID
 import scala.concurrent.{blocking, ExecutionContext, Future}
+import scala.util.{Failure, Success, Try}
 
 /**
  * This class knows how to validate Rawls billing project requests and instantiate linked billing profiles in the
  * billing profile manager service.
  */
 class BpmBillingProjectLifecycle(
-  billingRepository: BillingRepository,
+  val samDAO: SamDAO,
+  val billingRepository: BillingRepository,
   billingProfileManagerDAO: BillingProfileManagerDAO,
-  workspaceManagerDAO: HttpWorkspaceManagerDAO,
+  workspaceManagerDAO: WorkspaceManagerDAO,
   resourceMonitorRecordDao: WorkspaceManagerResourceMonitorRecordDao
 )(implicit val executionContext: ExecutionContext)
-    extends BillingProjectLifecycle
-    with LazyLogging {
```

Comment:
This still uses `LazyLogging`: it just gets it by way of `BillingProjectLifecycle`, where it was needed for `unregisterBillingProject`